app/src/main/java/org/greenstand/android/TreeTracker/activities/CameraActivity.kt

```diff
@@ -60,6 +62,9 @@ class CameraActivity : Activity(), Camera.PictureCallback, View.OnClickListener,
         intent.extras?.let {
             captureSelfie = intent.extras!!.getBoolean(ValueHelper.TAKE_SELFIE_EXTRA, false)
         }
+
+        setSupportActionBar(camera_toolbar)
+        supportActionBar?.title = "Take a Selfie"
```

Comment:
Can we move this to a string resource? Better yet, just add this as a string resource to the xml.

Reply:
Right, I actually did that in the XML.  Thanks.